src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public static User findByEmail(String email) {
+        for (User u : users) {
```

Comment:
```suggestion
        for (User user : users) {
```
avoid short one-letter naming, readability is important